### DISCUSSION TITLE
How to change VRAM

### DISCUSSION THREAD

**Discussion by mas0602** [lllyasviel/Fooocus#1381] [Q&A]
How can I change vram? I've already looked through other comments and came across the following: \Fooocus\backend\headless\fcbh\model_management.py but I can't find this path (backend\...)...

Pc Specs
Rx 5700xt
Win 10
16GB RAM

**Comment by Laurent-VueJS** [comment]
windows 10 manages your VRAM. It will allow physical VRAM (from your 5700xt) + maximum 50% of the RAM as virtual VRAM. see https://devblogs.microsoft.com/directx/gpus-in-the-task-manager/ You can see in performance manager the max VRAM and the usage of it. I think your config is a bit short for Fooocus on GPU as it often eats up more than 40GB of VRAM to run. You should consider running on CPU (with a boost of 40 or 50 GB on the virtual memory Control Panel > System > Advanced > Performance > Settings > Advanced > Virtual Memory)

**Reply by Laurent-VueJS** [reply]
addendum : the high VRAM usage seems linked with a bug (memory leak). So there is some hope it will be solved.
NB : it seems that to use an old version of Fooocus can be a temporary solution.

**Reply by mashb1t** [reply]
Please make sure you're using the latest version of Fooocus, which supports AMD GPUs with min. 8GB VRAM :)

**Reply by Laurent-VueJS** [reply]
yes I do have the latest version (version downloaded 09-jan-2024 and I have 16GB VRAM (48 GB RAM 68 GB swap).
I installed 5 times (original + some experiments to change some pyhton packages etc. but it did not succeed yet).nb : --always-cpu worls perfectly

**Reply by lgwjames** [reply]
im trying to use my amd card on my imac but takes 40 mins to do anything?

**Reply by mashb1t** [reply]
@lgwjames see https://github.com/lllyasviel/Fooocus/issues/2057#issuecomment-1912789314
